samples/hosting/aws-lambda-sqs/sample.md

```diff
@@ -22,50 +22,58 @@ The [`Amazon.Lambda.Tools` CLI](https://github.com/aws/aws-lambda-dotnet) can be
+2. Update the values of `stack-name` and `s3-bucket` settings in aws-lambda-tools-defaults.json file found in the **ServerlessEndpoint** project
 
 ## Running the sample
 
 INFO: It is not possible at this stage to use the AWS .NET Mock Lambda Test Tool to run the sample locally.
 
-Run the following command from the `AwsLambda.SQSTrigger` directory to deploy the Lambda project:
+Run the following command from the `ServerlessEndpoint` directory to deploy the Lambda project:
 
 `dotnet lambda deploy-serverless`
 
 The deployment will ask for a stack name and an S3 bucket name to deploy the serverless stack.
 
 After that, running the sample will launch a single console window:
 
-* **AWSLambda.Sender** is a console application that will send a `TriggerMessage` to the `AwsLambdaSQSTrigger` queue, which is monitored by the AWS Lambda.
-* The deployed **AWSLambda.SQSTrigger** project will receive messages from the `AwsLambdaSQSTrigger` queue and process them using the AWS Lambda runtime.
+* **RegularEndpoint** is a console application that will send a `TriggerMessage` to the `ServerlessEndpoint` queue, which is monitored by the AWS Lambda.
+* The deployed **ServerlessEndpoint** project will receive messages from the `ServerlessEndpoint` queue and process them using the AWS Lambda runtime.
 
-To try the AWS Lambda:
+To try the AWS Lambda
 
-1. From the **AwsLambda.Sender** window, press <kbd>Enter</kbd> to send a `TriggerMessage` to the trigger queue.
-1. The AWS Lambda will receive the `TriggerMessage` and process it with NServiceBus.
-1. The NServiceBus message handler for `TriggerMessage` sends a `FollowUpMessage`.
-1. The AWS Lambda will receive the `FollowUpMessage` and process it with NServiceBus.
-1. The NServiceBus message handler for `FollowUpMessage` sends a `BackToSenderMessage` that will be handled by the **AwsLambda.Sender**
+1. From the **RegularEndpoint** window, press <kbd>Enter</kbd> to send a `TriggerMessage` to the ServerLessEndpoint queue.
+1. The AWS Lambda will receive the `TriggerMessage` and hand off its procesing to NServiceBus.
+1. The NServiceBus message handler for `TriggerMessage` on **ServerlessEndpoint** sends a `ResponseMessage` that will be handled by the **RegularEndpoint**
 
 ## Code walk-through
 
 The static NServiceBus endpoint must be configured using details that come from the AWS Lambda `ILambdaContext`. Since that is not available until a message is handled by the function, the NServiceBus endpoint instance is deferred until the first message is processed, using a lambda expression such as:
 
 snippet: EndpointSetup
 
-The same class defines the AWS Lambda, which makes up the hosting for the NServiceBus endpoint. The `FunctionHandler` method hands off processing of the message to NServiceBus:
+The same class defines the AWS Lambda, which makes up the hosting for the NServiceBus endpoint. The `SqsHandler` method hands off processing of messages to NServiceBus:
```

Comment:
```suggestion
The same class defines the AWS Lambda, which makes up the hosting for the NServiceBus endpoint. The `FunctionHandler` method hands off processing of messages to NServiceBus:
```

Should we keep this as `FunctionHandler` since that is the actual name of the method?